README.md

```diff
@@ -36,14 +34,21 @@ This file is simple and self explanatory. The small example below is enough to u
             'some-casperjs-browsing.js': 'folder/script.js'
     ]
 
-Usage
------
+## Usage
 
 `phantombuster [-c config.cson] [script.coffee [other.coffee...]]`
 
-* The most typical usage is to watch for file modification while coding. Simply execute `phantombuster` in a directory containing `phantombuster.cson`.
-* Specify a different file than `phantombuster.cson`: `phantombuster -c config.cson`
-* Upload a specific script to Phantombuster (without watching): `phantombuster project/script.coffee` (must be a value in one of the `scripts` objects in `phantombuster.cson`)
-* Update your whole project: `phantombuster project/*.coffee`
+- The most typical usage is to watch for file modification while coding. Simply execute `phantombuster` in a directory containing `phantombuster.cson`.
+- Specify a different file than `phantombuster.cson`: `phantombuster -c config.cson`
+- Upload a specific script to Phantombuster (without watching): `phantombuster project/script.coffee` (must be a value in one of the `scripts` objects in `phantombuster.cson`)
+- Update your whole project: `phantombuster project/*.coffee`
+
+## Troubleshooting
+
+On MacOS, you might get an error when running the `prepublish` script. This is because of the `sed` command. To run it locally, install gsed (`brew install gnu-sed`) and replace the `publish:` script with:
+
+```
+"prepublish": "coffee --bare -o lib/ -c src/*.coffee && gsed -i'' -e '1s/^/\\#\\!\\/usr\\/bin\\/env node\\n/' lib/sdk.js",
+```
```

Comment:
This is a public repository, primary readers of this doc will be users of the SDK, not developers of the repo. This should either be in a separate file, or in a specific section for developers of the project.
I also submitted [a PR with a simpler suggestion for this issue](https://github.com/phantombuster/sdk/pull/9).